### PR TITLE
Fix style regressions

### LINKED ISSUE #670: Style regressions introduced
In #503 some style regressions were introduced;

*  The "Add Layers" button is now slightly larger and the top padding is off, in all browsers.
* The Tour modal has extra vertical padding added.

### LINKED ISSUE #671: Invalid attribute in meta viewport tag
index.html has a meta viewport tag with a `minimal-ui` attribute. Safari complains that this attribute is invalid.

### PR DESCRIPTION
**Pull request by ZachTRice**
Fixes #670, Fixes #671 

Changes in this pull request:
- Add "Segoe UI" default font back to "Add Layers" button
- Change tour step 1 to be a wide window
- Add `bordered` styles to tour windows which were removed from our version of joyride; 
this requires using an after element on the `nub` element whereas before there was a second element.
- Fix margins in tour windows
- Remove `minimal-ui` viewport attribute

### CODE REVIEW

File: web/css/jquery-ui-override.css [hunk 1]
```diff
@@ -1,6 +1,10 @@
 /*
  * Default jquery-ui button
  */
+.ui-button-text {
+  font-family: "Segoe UI", "Arial", sans-serif;
```
Comment (localjo) [inline web/css/jquery-ui-override.css]: It looks like the original "Segoe UI" declaration is still there, it's just overridden by [these two lines](https://github.com/nasa-gibs/worldview/blob/module-loaders/web/css/sidebar.css#L45-L46) added in https://github.com/nasa-gibs/worldview/commit/8345cb170710cbe9d5dfaa07358a8cdbb7a18c55 that we could probably remove to be more efficient.

File: web/css/tour.css [hunk 2]
```diff
@@ -431,34 +438,79 @@ p.wv-tour-page-count {
...
 }
 
+.joyride-tip-guide.toolbarTip span.joyride-nub.top {
+  left: 250px;
+}
+
+.joyride-tip-guide.bordered span.joyride-nub.top::after {
+  top: -16px;
+  left: -15px;
+  border: 15px solid #808080 !important;
```
Comment (localjo) [inline web/css/tour.css]: Are all of these `!important` declarations necessary? These are pretty specific selectors already.

File: web/pages/tour.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <p class="splashwelcome">
       This application allows you to interactively browse global satellite imagery within hours of it being acquired. Use the features described below to find interesting imagery, save and share what you find, and download the underlying data.
     </p>
-    <br>
+    <br><br>
```
Comment (localjo) [inline web/pages/tour.html]: I'm not quite sure what caused the extra spacing in production, but I think a single space is fine, and fits the rhythm of the page better. A double `<br><br>` seems unnecessary. What do you think about removing it?

Comment (ZachTRice) [reply web/pages/tour.html]: I agree, I added it back in for parity with what we had before.

File: web/css/tour.css (hunk 2 above)
Comment (ZachTRice) [reply web/css/tour.css]: Actually, that's a good point. On the after elements, `!important` should not be needed. I will double-check and remove.

**Pull request merged**